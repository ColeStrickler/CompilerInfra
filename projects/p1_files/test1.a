+-
"swag";
yolo = "swag";
123243443
"swagmoneyballa"
kingyolo = "swag"; + * 3132 * sas money_yolo, swag; x++
meme-- lol02_swag--
{ meme; 123++ };
while{
    truck;
    swag = 2 + ( { }) / a -> nUm_ == > >= <= 4
}