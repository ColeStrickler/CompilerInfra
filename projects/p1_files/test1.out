CROSS [1,1]
DASH [1,2]
STRINGLITERAL:"swag" [2,1]
SEMICOL [2,7]
ID:yolo [3,1]
ASSIGN [3,6]
STRINGLITERAL:"swag" [3,8]
SEMICOL [3,14]
INTLITERAL:123243443 [4,1]
STRINGLITERAL:"swagmoneyballa" [5,1]
ID:kingyolo [6,1]
ASSIGN [6,10]
STRINGLITERAL:"swag" [6,12]
SEMICOL [6,18]
CROSS [6,20]
STAR [6,22]
INTLITERAL:3132 [6,24]
STAR [6,29]
ID:sas [6,31]
ID:money_yolo [6,35]
COMMA [6,45]
ID:swag [6,47]
SEMICOL [6,51]
ID:x [6,53]
POSTINC [6,54]
ID:meme [7,1]
POSTDEC [7,5]
ID:lol02_swag [7,8]
POSTDEC [7,18]
LCURLY [8,1]
ID:meme [8,3]
SEMICOL [8,7]
INTLITERAL:123 [8,9]
POSTINC [8,12]
RCURLY [8,15]
SEMICOL [8,16]
WHILE [10,1]
LCURLY [10,6]
ID:truck [11,5]
SEMICOL [11,10]
ID:as [11,12]
ID:swag [12,5]
ASSIGN [12,10]
INTLITERAL:2 [12,12]
CROSS [12,14]
LPAREN [12,16]
LCURLY [12,18]
RCURLY [12,20]
RPAREN [12,21]
SLASH [12,23]
ID:a [12,25]
ARROW [12,27]
ID:nUm_ [12,30]
EQUALS [12,35]
GREATER [12,38]
GREATEREQ [12,40]
LESSEQ [12,43]
INTLITERAL:4 [12,46]
RCURLY [13,1]
REF [13,3]
ID:x [13,4]
SEMICOL [13,5]
LESS [13,7]
INTLITERAL:1 [13,8]
ID:asad [13,9]
ID:y [18,2]
ID:olo [18,4]
ID:meme [18,8]
INTLITERAL:1214212421 [21,1]
SEMICOL [21,11]
EOF [21,12]
