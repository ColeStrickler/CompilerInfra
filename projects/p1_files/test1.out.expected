CROSS [1,1]
DASH [1,2]
STRINGLITERAL:"swag" [2,1]
SEMICOL [2,7]
ID:yolo [3,1]
ASSIGN [3,6]
STRINGLITERAL:"swag" [3,8]
SEMICOL [3,14]
INTLITERAL:123243443 [4,1]
STRINGLITERAL:"swagmoneyballa" [5,1]
ID:kingyolo [6,1]
ASSIGN [6,10]
STRINGLITERAL:"swag" [6,12]
SEMICOL [6,18]
CROSS [6,20]
STAR [6,22]
INTLITERAL:3132 [6,24]
STAR [6,29]
ID:sas [6,31]
ID:money_yolo [6,35]
COMMA [6,45]
ID:swag [6,47]
SEMICOL [6,51]
ID:x [6,53]
POSTINC [6,54]
ID:meme [7,1]
POSTDEC [7,5]
ID:lol02_swag [7,8]
POSTDEC [7,18]
LCURLY [8,1]
ID:meme [8,3]
SEMICOL [8,7]
INTLITERAL:123 [8,9]
POSTINC [8,12]
RCURLY [8,15]
SEMICOL [8,16]
WHILE [9,1]
LCURLY [9,6]
ID:truck [10,5]
SEMICOL [10,10]
ID:swag [11,5]
ASSIGN [11,10]
INTLITERAL:2 [11,12]
CROSS [11,14]
LPAREN [11,16]
LCURLY [11,18]
RCURLY [11,20]
RPAREN [11,21]
SLASH [11,23]
ID:a [11,25]
ARROW [11,27]
ID:nUm_ [11,30]
EQUALS [11,35]
GREATER [11,38]
GREATEREQ [11,40]
LESSEQ [11,43]
INTLITERAL:4 [11,46]
RCURLY [12,1]
EOF [12,2]
